docs/web-wallet.md

```diff
@@ -170,61 +170,17 @@ const verificationRequest = {
 const base64EncodedVerificationRequest = btoa(JSON.stringify(verificationRequest));
 
 // Open the Polygon ID Verification Web Wallet with the encoded verification request
-window.open(`https://web-wallet-test.polygonid.me/#${base64EncodedVerificationRequest}`);
-```
-
-</TabItem>
-
- <TabItem value="On-Chain Verification">
-
-```js
-// Define the verification request
-const verificationRequest = {
-  backUrl: "https://my-app.org/back",
-  finishUrl: "https://my-app.org/finish",
-  logoUrl: "https://my-app.org/logo.png",
-  name: "My app",
-  zkQueries: [
-    {
-      circuitId: "credentialAtomicQuerySigV2OnChain",
-      id: 1713976575,
-      query: {
-        allowedIssuers: ["*"],
-        context:
-          "https://raw.githubusercontent.com/anima-protocol/claims-polygonid/main/schemas/json-ld/pol-v1.json-ld",
-        type: "AnimaProofOfLife",
-        credentialSubject: {
-          human: {
-            $eq: true,
-          },
-        },
-      },
-    },
-  ],
-  verifierDid: "did:polygonid:polygon:amoy:2qV9QXdhXXmN5sKjN1YueMjxgRbnJcEGK2kGpvk3cq",
-  transactionData: {
-    contractAddress: "0x62811c9e1C8b2397767779BC8ff5Ca48869a61Fc",
-    functionName: "submitZKPResponse",
-    methodId: "0xb68967e2",
-    chainId: 80002,
-    network: "polygon-amoy",
-  },
-};
-// Encode the verification request to base64
-const base64EncodedVerificationRequest = btoa(JSON.stringify(verificationRequest));
-
-// Open the Polygon ID Verification Web Wallet with the encoded verification request
-window.open(`https://web-wallet-test.polygonid.me/#${base64EncodedVerificationRequest}`);
+window.open(`https://wallet.privado.id/#${base64EncodedVerificationRequest}`);
```

Comment:
Just comment out the onchain part, you would need to bring it back soon.

Reply:
Yes i have commented out the whole on-chain part

Reply:
Will bring it back onec we have it working